lib/libzrepl/data_conn.c

```diff
@@ -334,6 +341,12 @@ uzfs_zvol_worker(void *arg)
 				}
 			}
 
+			/* App IOs should go to cloen_zv */
```

Comment:
we can make this as 'else' for above 'if' without check for clone_zv

Reply:
done